### PR TITLE
carbon management

### PR DESCRIPTION
**Pull request by fneum**
## Geological Sequestration Potentials

See https://github.com/ericzhou571/Co2Storage

For required file `complete_map_2020_unit_Mt.geojson`, checkout:

https://raw.githubusercontent.com/ericzhou571/Co2Storage/main/resources/complete_map_2020_unit_Mt.geojson

Still needs a few configuration options moved to `config.yaml`.

### CODE REVIEW

File: config.default.yaml [hunk 1]
```diff
@@ -250,9 +250,11 @@ sector:
   dac: true
   co2_vent: false
   SMR: true
+  sequestration_potential: true  # geological co2 storage potential
   co2_sequestration_potential: 200  #MtCO2/a sequestration potential for Europe
   co2_sequestration_cost: 10   #EUR/tCO2 for sequestration of CO2
-  co2_network: false
+  co2_spatial: false
+  co2network: false
```
[the remark this answers is not in the document]
Comment (fneum) [reply config.default.yaml:257]: yes, otherwise the `{sector_opts}` wildcard does not work as `_` is used as separators between wildcards.

**Comment by fneum** [comment]
@FabianHofmann, what's your view? Merge before or after this week's release?

**Comment by FabianHofmann** [comment]
It looks stable, I think we can merge it now

File: scripts/helper.py [hunk 2]
```diff
@@ -138,6 +138,6 @@ def parse(l):
 
 def update_config_with_sector_opts(config, sector_opts):
     for o in sector_opts.split("-"):
-        if o.startswith("CF:"):
+        if o.startswith("CF+"):
```
Comment (FabianHofmann) [inline scripts/helper.py:141]: This was the only thing I was wondering about

Comment (fneum) [reply scripts/helper.py:141]: See the discussion at https://github.com/PyPSA/pypsa-eur-sec/pull/251. 

This should have been "+" already.

**Pull request merged**